src/main/java/org/launchcode/PetLife/models/PetBio.java

```diff
@@ -0,0 +1,150 @@
+package org.launchcode.PetLife.models;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.HashMap;
+
+
+@Entity
+//@Table(name="pet_bio")
+public class PetBio<petGender> extends AbstractEntity{
+
+    //@ManyToOne
+//    @JoinColumn(name = "petbio_id")
+
+
+    @NotBlank(message = "Name is required")
+    @Size(min = 1, max = 50, message = "Name must be between 3 and 50 characters")
+    //@Column (name="pet_name")
+    private String petName;
+
+//@Column (name="pet_age")
+    @NotNull(message = "Age is required")
+     private Integer petAge;
+
+//
+//   // @Column (name="pet_species")
+@NotNull(message = "Species is required")
+    private String petSpecies;
+//
+//    //@Column (name="pet_breed")
+@NotNull(message = "Breed is required or input unknown")
```

Comment:
maybe we could add a limit of characters in case people input so many characters.